openmetadata-ui/src/main/resources/ui/src/components/GlobalSetting/GlobalSetting.less

```diff
@@ -74,3 +72,16 @@
 .side-panel-icons {
   color: @icon-color;
 }
+
+.global-settings-left-panel-container {
+  height: calc(100vh - 90px);
+  border: 1px solid @left-panel-border-color;
+  border-radius: 4px;
+  box-shadow: 1px 1px 8px @box-shadow-color;
+  overflow-y: scroll;
+  -ms-overflow-style: none;
+  scrollbar-width: none;
+}
```

Comment:
We have css for this panel under this class: `global-setting-left-panel`

Reply:
made a global styling for class left-panel-container and removed this